src/pages-and-resources/progress/Settings.jsx

```diff
@@ -26,17 +26,23 @@ function ProgressSettings({ intl, onClose }) {
       onSettingsSave={handleSettingsSave}
     >
       {
-        ({ handleChange, handleBlur, values }) => (
-          <FormSwitchGroup
-            id="enable-progress-graph"
-            name="enableProgressGraph"
-            label={intl.formatMessage(messages.enableGraphLabel)}
-            helpText={intl.formatMessage(messages.enableGraphHelp)}
-            onChange={handleChange}
-            onBlur={handleBlur}
-            checked={values.enableProgressGraph}
-          />
-        )
+        ({ handleChange, handleBlur, values }) => {
+          let formSwitch = null;
+          if (process.env.ENABLE_PROGRESS_GRAPH_SETTINGS === 'true') {
```

Comment:
You should merge this setting with the others in main index file inside `mergeConfig` and then you can load it here from the AppContext or using `getConfig`.

Reply:
Thanks for the suggestion, I have updated the code.